package.json

```diff
@@ -41,6 +41,7 @@
     "@types/tar-stream": "^1.6.0",
     "chai": "^4.1.2",
     "chai-as-promised": "^7.1.1",
+    "husky": "^3.0.9",
```

Comment:
@CameronDiver, do you know / remember if `husky` is a dev or production dependency for livepush? This PR assumes it's a dev dependency, but if not, then I should drop this PR and amend `balena-cli's `package.json` instead.